factory/etc/systemd/bootchart.conf

```diff
@@ -0,0 +1,8 @@
+[Bootchart]
+Samples=36000
+Frequency=20
```

Comment:
so this should run for 30 minutes? that seems excessive for the initrd, no? The man page doesn't actually specify what units the Frequency is in, so I'm assuming it's Hertz, so 36000 samples / ( 20 samples / second) == 1800 seconds == 30 minutes

Reply:
30 minutes is the maximum time. Actually, the daemon will be killed much earlier when systemd switches root, using SIGHUP for that as we specify in the unit. I've just put a big number to make sure we capture all data in extreme cases.

Reply:
Right, I missed that the bootchart will be stopped when we switch root with SIGHUP. This is fine then.